appeals/e2e/cypress/support/appealsApiClient.js

```diff
@@ -385,6 +385,38 @@ export const appealsApiClient = {
 		}
 	},
 
+	async reviewStatement(reference) {
```

Comment:
Instead of creating a new endpoint, we should use the existing POST /api/appeals/representation-submission.